core/core.go

```diff
@@ -104,6 +105,7 @@ type IpfsNode struct {
 	Diagnostics  *diag.Diagnostics   // the diagnostics service
 	Ping         *ping.PingService
 	Reprovider   *rp.Reprovider // the value reprovider system
+	IpnsRepub    *ipnsrp.Republisher
```

Comment:
We'll eventually want to organize this stuff cleaner. maybe bundle all the IPNS parts into one struct.